### ISSUE TITLE
Add support for the new GitHub README template.

### ISSUE THREAD

**Issue by ghost** [wikimatze/hammer.vim#39]
I'd love the new GitHub README template to be available as the new default.
The current default could be renamed `github_old`.

**Comment by wikimatze** [comment]
Yeah, I'm working on it, and will release it at the end of the week.

**Comment by wikimatze** [comment]
It's done - see commit https://github.com/matthias-guenther/hammer.vim/commit/078cb118b156f46be01c5dedf80ec3aaeae3f283

**Comment by ghost** [comment]
Thanks! I think it might need to be tweaked, though, because the font is not the same and the padding on &lt;li&gt; elements seems to be different as well. I think it could be cool if we wrapped the README in the same UI element that GItHub is using. Thoughts?

**Comment by wikimatze** [comment]
See commits https://github.com/matthias-guenther/hammer.vim/compare/b67eed84a7...eb1cec166d - in my eyes it should fit now.

**Comment by wikimatze** [comment]
Is the layout okay for now or should I ask the github people for their layout?

**Comment by ghost** [comment]
I think the layout is closer but not close enough. The original README template(old github theme) was _almost_ identical, and I'd like the new template to have the same level of similarity. I don't think we need to ask GitHub for their markup, I think the layout can be easily matched (we're aiming for 95%+ similarity, not 100%). After that, we just need to apply similar CSS rules to make sure everything matches. 

I see the old github theme is totally gone as well, I think we should keep it around, maybe as `github-legacy`.

**Comment by wikimatze** [comment]
`github-legacy` is a good idea - I will restore it this evening.
